Tests/Spec/tentacle_scenario_01_install_spec.rb

```diff
@@ -136,6 +136,40 @@
   its(:content) { should match /Tentacle\.Communication\.TrustedOctopusServers.*#{ENV['OctopusServerThumbprint']}/}
 end
 
+### worker tentacle
+
+describe service('OctopusDeploy Tentacle: WorkerTentacle') do
+  it { should be_installed }
+  it { should be_running }
+  it { should have_start_mode('Automatic') }
+  it { should run_under_account('LocalSystem') }
+end
+
+describe port(10937) do
+  it { should be_listening.with('tcp') }
+end
+
+describe octopus_deploy_tentacle(ENV['OctopusServerUrl'], ENV['OctopusApiKey'], "WorkerTentacle") do
+  it { should exist }
+  it { should be_registered_with_the_server }
+  it { should be_online }
+  it { should be_listening_tentacle }
+  it { should be_in_environment('The-Env') }
+  it { should have_role('Test-Tentacle') }
+  it { should have_display_name("WorkerTentacle")}
+  # TODO check pool membership
+end
+
+describe windows_registry_key('HKEY_LOCAL_MACHINE\Software\Octopus\Tentacle\WorkerTentacle') do
```

Comment:
This wont exist will it? It's a new shiny tentacle, so will only have the file in c:\programdata, right?

Reply:
Yep, I think you're correct